### DISCUSSION TITLE
Rofi 1.7.5 doesn't allow me to add flags to run commands.

### DISCUSSION THREAD

**Discussion by rascencio-kraft** [davatorium/rofi#1737] [(Possible) Issues]
For example, typing `firefox -p` with `run` only executes `firefox` and ignores the arguments, it used to do it before, do we need to add or change settings to have it work like that again?

**Comment by DaveDavenport** [comment]
it executes the selected entry, not what you entered. 
Many years ago (2017) we added the option to negate matches, so `firefox -p` matches everything that has firefox and not `p`, therefore it matches firefox.

Try hitting the 'custom entry' keybinding  (Control+Return by default) to launch what you entered instead of what you matched.
